docs/source/index.rst

```diff
@@ -6,10 +6,10 @@
 Welcome to Flow
 ================
 
-Flow is a computational framework for deep RL and control experiments for traffic microsimulation.
+`Flow <https://berkeleyflow.github.io>`_ is a computational framework for deep RL and control experiments for traffic microsimulation. Visit `our website <https://berkeleyflow.github.io>`_ for more information. 
 
 Flow is a work in progress - input is welcome. Available documentation is limited for now. 
-Tutorials in iPython notebook format are available through `our GitHub <https://github.com/berkeleyflow/flow/tree/master/tutorials/exercises>`_.
+Tutorials `are available <https://github.com/berkeleyflow/flow/tree/master/tutorials/exercises>`_ in iPython notebook format.
```

Comment:
I like the new text! Can we hyperlink "Tutorials" instead?

Reply:
Addressed.